routing/RoutingEngine.h

```diff
@@ -127,8 +112,8 @@ class RoutingEngine {
 private:
      /// collections of all routers used
      std::vector<Router*> _routersCollection;
-     /// collection of all trips/routes
-     Trips _tripsCollection;
+
+     /// states if the routers need to be updated
```

Comment:
I think the filed name is already very good and does not need additional explanation with a comment